complex/ft_comp.c

```diff
@@ -54,8 +54,6 @@ static int ft_open_cntrs(void)
 {
 	int ret;
 
-	opts.comp_method = test_info.cq_wait_obj;
-
```

Comment:
The assignment was done there because I was trying to reuse the ft_cntr_open call in common/shared.c but didn't realize it used a different enum. In order to actually support the cq_wait_obj defined in the configuration, the cq_wait_obj needs to be set. The proper fix in order to support it would be to call fi_cntr_open instead of ft_cntr_open, passing in the needed cntr_attr.
This fix shouldn't cause any failures since it would just default to WAIT_NONE, but it wouldn't be testing it correctly.